apps/jobs/urls.py

```diff
@@ -110,6 +110,11 @@
         views.update_submission_started_at,
         name="update_submission_started_at",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/submissions/(?P<submission_pk>[0-9]+)/cancel/$",
```

Comment:
Why do we need `challenge_pk` here? I think `submission_pk` should be enough here, no?

Reply:
We only want the participants of the team to cancel the submission using UI. To check whether team has participated in the challenge we need challenge pk